deep_quoridor/src/agents/dexp.py

```diff
@@ -46,20 +46,21 @@ def forward(self, x):
 
 
 @dataclass
-class DExpPlayParams(TrainableAgentParams):
+class DExpAgentParams(TrainableAgentParams):
+    # Indicates whether to rotate the board for player_1
     rotate: bool = False
+    # Indicates whether to include the turn information in tensor
     turn: bool = False
+    # Indicates whether to split the board into two channels (one for each player)
     split: bool = False
 
-    @classmethod
-    def from_str(cls, agent_params_str="000"):
-        p = DExpPlayParams(
-            use_rotate_board=bool(int(agent_params_str[0])),
-            include_turn=bool(int(agent_params_str[1])),
-            split_board=bool(int(agent_params_str[2])),
-        )
-        return p
+    # Whether oppoments actions are used for training
+    # This is used for training only, not for playing
+    use_opponents_actions = False
 
+    # Parameters used for training which are required to be used with the same set of values during training
+    #  and playing are used to generate a 'key' to identify the model.
+    # This is used to load the model from disk or wandb
```

Comment:
Unless you changed it and I missed it, we don't use it to load the model from wandb.  In wandb, we just load a specific model based on the alias and then we figure out the parameters from the metadata